literature.bib

```diff
@@ -250,6 +250,16 @@ @inproceedings{BKH+:VaMoS24
 	pages = {27--36}
 }
 
+@dataset{BKH+:VaMoS24-Artifact,
+	aauthor = {B\"{o}hm, Sabrina and Krieter, Sebastian and He\ss{}, Tobias and Th\"{u}m, Thomas and Lochau, Malte},
```

Comment:
Use the `test.sh` script
It detects the bibtex incompatible `@dataset` tag and the missing `author` field